components/layout_2020/flexbox/layout.rs

```diff
@@ -1309,7 +1309,7 @@ fn do_initial_flex_line_layout<'items>(
 
 impl InitialFlexLineLayout<'_> {
     fn new<'items>(
-        flex_context: &mut FlexContext,
+        flex_context: &FlexContext,
```

Comment:
Should this be part of this PR?

Reply:
Nope. This wasn't meant to be part of this PR. I've removed it.